test/EndToEnd/GivenUsingDefaultRuntimeFrameworkVersions.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.TestFramework;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using NuGet.Versioning;
+using Xunit;
+
+namespace EndToEnd
+{
+    public partial class GivenUsingDefaultRuntimeFrameworkVersions : TestBase
+    {
+        private static readonly IEnumerable<string> frameworks = new string[] {"Microsoft.NETCore.App", "Microsoft.WindowsDesktop.App",
+            "Microsoft.WindowsDesktop.App.WPF", "Microsoft.WindowsDesktop.App.WindowsForms", "Microsoft.AspNetCore.App" };
+
+        private static readonly IEnumerable<string> versions = SupportedNetCoreAppVersions.Versions.Where(version => NuGetVersion.Parse(version).Major >= 3);
+
+        [Fact]
+        public void DefaultRuntimeVersionsAreUpToDate()
+        {
+            var directory = TestAssets.CreateTestDirectory();
+            string projectDirectory = directory.FullName;
+            var outputFile = "resolvedVersions.txt";
+
+            new NewCommandShim()
+                .WithWorkingDirectory(projectDirectory)
+                .Execute("console --no-restore")
+                .Should().Pass();
+
+            var projectFile = new DirectoryInfo(projectDirectory).GetFiles("*.csproj").First().FullName;
+            var project = XDocument.Load(projectFile);
+            var ns = project.Root.Name.Namespace;
+            string writeResolvedVersionsTarget = @$"
```

Comment:
See if you can use https://github.com/dotnet/sdk/blob/ab2176b3447e3e7820b5a340408110b6d498f907/src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToSetPropertiesInDirectoryBuildProps.cs#L67

to replace these

Reply:
It doesn't look like there's an equivalent `GetValuesCommand` in the testing setup for this repo, but under the hood it seems to do essentially the same thing we're doing here anyways.